cogeo_mosaic/backends/stac.py

```diff
@@ -84,6 +91,13 @@ def __attrs_post_init__(self):
         self.mosaic_def = self._read()
         self.bounds = self.mosaic_def.bounds
 
+        mosaic_tms = self.mosaic_def.tilematrixset or WEB_MERCATOR_TMS
+
+        # By mosaic definition the bounds and CRS are defined using the TMS
+        # Geographic CRS.
+        self.crs = mosaic_tms.rasterio_geographic_crs
```

Comment:
think this is a typo, as both crss are the geographic_crs here

Reply:
> as both crss are the geographic_crs here

Yes. 
By definition the `bounds` property defined in the mosaic document use WGS84 (or geographic CRS) while in rio-tiler when using any dataset the `bounds` is using the dataset's `crs` which is why we needed a `geographic_crs` property 

FYI: this is used there https://github.com/cogeotiff/rio-tiler/blob/6d71e7d380cd815b8f65a1917653769d882c2aa6/rio_tiler/io/base.py#L90-L108

Reply:
I understand this is confusing but it comes from the mosaicjson definition 😓